client/lib/plans/plans-list.js

```diff
@@ -794,8 +802,16 @@ export const PLANS_LIST = {
 			),
 		getDescription: () =>
 			i18n.translate(
-				'Comprehensive, automated scanning for security vulnerabilities, ' +
-					'fast video hosting, and marketing automation.'
+				'{{strong}}Best for Small Businesses:{{/strong}}' +
+					'Comprehensive, automated scanning for security vulnerabilities, ' +
+					'fast video hosting, and marketing automation.',
+				{
+					components: {
```

Comment:
Looks like we are repeating this component in a bunch of places. 
We can make it a constant and pass in the constant into the `i18n.translate( text, translationConstant )`

where
```
const translationComponents = {
					components: {
Attach files by dragging & dropping, selecting or pasting them.
						strong: (
							<strong className="plans__features plan-features__targeted-description-heading" />
						),
					};
```